README.md

```diff
@@ -1 +1,108 @@
-# sm-server
\ No newline at end of file
+# sm-server
+Sm-server is an HTTP server that provides sm algorithm services
+
+# Quick start
+```shell
+$ git clone git@github.com:cryptape/sm-server.git
+$ PORT=8888 cargo run
```

Comment:
I recommend using command line arguments instead of environment variables. Second, teach users how to compile the `release` version and `musl` version.